### PR TITLE
Changes for WB compatibility

### PR DESCRIPTION
**Pull request by shishir-msft**

### CODE REVIEW

File: migration/azuredeploy-migration.json [hunk 1]
```diff
@@ -87,20 +60,6 @@
...
-            },
-            "type": "string"
-        },
-        "autoscaleVmCountMax": {
-            "defaultValue": 10,
-            "metadata": {
-                "description": "Maximum number of autoscaled web VMs"
-            },
-            "type": "int"
-        },
```
Comment (venu-sivanadham) [inline migration/azuredeploy-migration.json:103]: Please retain autoscaleVmSku input. Allowing customers to change the target infra VM SKU is in scope of current release.

File: migration/azuredeploy-migration.json [hunk 2]
```diff
@@ -2,26 +2,12 @@
...
-            "defaultValue": ""
-        },
         "redisDeploySwitch": {
-            "defaultValue": false,
+            "defaultValue": "false",
             "metadata": {
                 "description": "Switch to deploy a redis cache or not. Note that certain versions of Moodle (e.g., 3.1) don't work well with Redis, so use this only for known well-working Moodle versions (e.g., 3.4)."
             },
-            "type": "bool"
+            "type": "string"
```
Comment (venu-sivanadham) [inline migration/azuredeploy-migration.json:14]: add "allowedValues": [ "true", "false" ] property for those input parameter converted from bool to string type.

Comment (shishir-msft) [reply migration/azuredeploy-migration.json:14]: Done, thanks.

File: migration/azuredeploy-migration.json (hunk 1 above)
Comment (shishir-msft) [reply migration/azuredeploy-migration.json:103]: Done, thanks.

**Pull request merged**